src/core/feed.factory.ts

```diff
@@ -46,8 +46,10 @@ export class FeedFactory {
   public blockedUsers(): BlockedUsersFeed {
     return new BlockedUsersFeed(this.client);
   }
-  public directInbox(): DirectInboxFeed {
-    return new DirectInboxFeed(this.client);
+  public directInbox(limit: number = 20): DirectInboxFeed {
```

Comment:
@Nerixyz what do you think, is that good idea to have `limit` as standalone argument, or will be better to wrap it in some `options: DirectInboxFeedOptions` object?

Reply:
> 
> 
> @Nerixyz what do you think, is that good idea to have `limit` as standalone argument, or will be better to wrap it in some `options: DirectInboxFeedOptions` object?

If other feeds will also accept the limit argument I'd better wrap in in another object to show consistency.